db/migrate/20200624162309_change_look_for_member_to_team.rb

```diff
@@ -0,0 +1,5 @@
+class ChangeLookForMemberToTeam < ActiveRecord::Migration[6.0]
+  def change
+    change_column :teams, :look_for_member, :boolean, :default => true, :null => true
```

Comment:
No need for 2 migrations, since this is not yet merged into master you can just run
```bash
bundle exec rake db:rollback STEPS=2
```
then delete your 2nd migration and fix the mistakes in your first migration file and then run
```bash
bundle exec rake db:migrate
```